### PR TITLE
Minor tweaks of button alignment and queue spacing

### PR DESCRIPTION
**Pull request by thornySoap**
<table>
<tr>
<td colspan=3>Before</td>
</tr>
<tr>
<td> 

player_before

</td>
<td>

queue_lower_before

</td>
<td>

queue_upper_before

</td>
</tr>
<tr>
<td colspan=3>After</td>
</tr>
<tr>
<tr>
<td>

player

</td>
<td>

queue_lower

</td>
<td>

queue_upper

</td
</tr>
</table>

To be honest, those things really bugged me, maybe I'm just weird D:

I'd appreciate some feedback, especially if I broke some conventions. Both the `IconButton` in `Player` and the `ResizableIconButton` in `Queue` were the only ones of their kind in their file, so I figured it would be more consistent to replace them with their counterpart.

### CODE REVIEW

File: app/src/main/res/drawable/bedtime.xml [hunk 1]
```diff
@@ -5,5 +5,5 @@
     android:viewportHeight="960">
     <path
```
Comment (thornySoap) [inline app/src/main/res/drawable/bedtime.xml:6]: Really weird that the icon itself wasn't aligned.
Just downloaded the vector asset again and it was centered

**Comment by mostafaalagamy** [comment]
When the repeat mode is activated in the queue, it doesn't show that it is activated.

**Comment by thornySoap** [comment]
@mostafaalagamy Thank you for testing! This should be resolved now. I hope ...

**Comment by mostafaalagamy** [comment]
> @mostafaalagamy Thank you for testing! This should be resolved now. I hope ...

It's resolved, but I think there's a conflict with the PR (unify multiselect look with playlist screen). Please review it.

**Comment by thornySoap** [comment]
That's possible. When I cherry picked the other commit to this branch there were no problems but I don't know what it'll look like when merging. But is there a way to resolve conflicts right now anyway? I think I have to wait until one of the PRs is (potentionally) merged and then I can fix the other one. But frankly, I don't know much about git ...

**Comment by mostafaalagamy** [comment]
> That's possible. When I cherry picked the other commit to this branch there were no problems but I don't know what it'll look like when merging. But is there a way to resolve conflicts right now anyway? I think I have to wait until one of the PRs is (potentionally) merged and then I can fix the other one. But frankly, I don't know much about git ...

I think after the merger it will be fix , thanks.

File: app/src/main/java/com/malopieds/innertune/ui/player/Queue.kt [hunk 2]
```diff
@@ -269,8 +267,8 @@ fun Queue(
                 WindowInsets.systemBars
                     .add(
                         WindowInsets(
-                            top = ListItemHeight,
-                            bottom = ListItemHeight,
+                            top = ListItemHeight + 12.dp + 8.dp,
+                            bottom = ListItemHeight + 8.dp,
```
Comment (Malopieds) [inline app/src/main/java/com/malopieds/innertune/ui/player/Queue.kt:271]: this is rather bad practice to "hardcode" values like this

Comment (thornySoap) [reply app/src/main/java/com/malopieds/innertune/ui/player/Queue.kt:271]: Yes, you're completely right, but I wasn't sure how it was handled in this app. In my own I always use dimension resources. But regarding the Queue maybe we could use Scaffold and put the upper and lower bars into the corresponding "topBar" and "bottomBar" letting the Scaffold measure their heights, so theres no need for a custom layout?

Comment (thornySoap) [reply app/src/main/java/com/malopieds/innertune/ui/player/Queue.kt:271]: But regarding the 8.dp spacing ... I mean its hardcoded all over the app:

Screenshot 2024-07-25 08:28:38

I personally favor dimension resources, but it doesn't seem to be used here and what would be the alternative?

**Comment by Malopieds** [comment]
Thanks a lot for this PR, sorry for the time it took for me to accept it, I had a lot of things going on!

> I personally favor dimension resources, but it doesn't seem to be used here and what would be the alternative?

I agree with you, it's how it should be handled, but right now I would gladly focus on fixing bugs/new features, but one day, that will be done !

**Pull request merged**